pkg/ebpf/c/tracee.bpf.c

```diff
@@ -7337,28 +7343,34 @@ CGROUP_SKB_HANDLE_FUNCTION(proto_icmpv6);
 // Network submission functions
 //
 
-#define FULL    1
-#define HEADERS 0
+#define FULL    65536 // 1 << 16
```

Comment:
1 << 16 is actually the max size of an IP packet. The sk_buff contains multiple IP packets (in the same TCP segment, when TCP is being used) but its max size is 64k as well. When setting this to 64k it would overflow the IPv4 packet length field, so I simply use the number to tell cgroup_skb_submit to send the ENTIRE packet (whenever 64k is used). This is done through flags argument of that function.